modules/grpc/otel/otel-protobuf-parser.cpp

```diff
@@ -1209,9 +1209,9 @@ syslogng::grpc::otel::ProtobufParser::set_syslog_ng_macros(LogMessage *msg, cons
 
       if (name.compare("PRI") == 0)
         {
-          if (!_value_case_equals(msg, macro, AnyValue::kBytesValue))
+          if (!_value_case_equals(msg, macro, AnyValue::kIntValue))
             continue;
-          msg->pri = log_rewrite_set_pri_convert_pri(macro.value().bytes_value().c_str());
+          msg->pri = macro.value().int_value();
```

Comment:
I think we should keep the bytes value parsing, too, for the sake of backward compatibility. We can check for int first, as it is the most probable.

(e.g. syslog-ng 4.6 sender -> syslog-ng 4.7 receiver)